spec/fixtures/hiera/clearinghouse_config.yaml

```diff
@@ -2,7 +2,7 @@
 :backends:
   - yaml
 :hierarchy:
-  - chipmunk
+  - clearinghouse
```

Comment:
Not sure why the clearinghouse hiera config was pointing at chipmunk, but fixed that.